frontend-react/src/pages/tos-sign/TermsOfServiceForm.tsx

```diff
@@ -14,7 +14,7 @@ import { Link } from "react-router-dom";
 import { Helmet } from "react-helmet";
 
 import Title from "../../components/Title";
-import { getStates } from "../../utils/OrganizationUtils";
+import getStates from "../../utils/States";
```

Comment:
"states" is an overused term in react. Plus, technically it's states and territories. I know it's a picky, but maybe rename to `StatesTerritories` ? 
I don't feel strongly about this naming either way, just feedback.